addons/web/static/src/views/fields/integer/integer_field.js

```diff
@@ -8,23 +8,38 @@ import { useInputField } from "../input_field_hook";
 import { standardFieldProps } from "../standard_field_props";
 import { useNumpadDecimal } from "../numpad_decimal_hook";
 
-import { Component } from "@odoo/owl";
+import { Component, useState } from "@odoo/owl";
 
 export class IntegerField extends Component {
     static template = "web.IntegerField";
     static props = {
         ...standardFieldProps,
         formatNumber: { type: Boolean, optional: true },
+        humanReadable: { type: Boolean, optional: true },
+        decimals: { type: Number, optional: true },
```

Comment:
I'm not sure "decimals" is a good name for an option for an integer field. I have No better proposal, thought.

Reply:
Hello, I have just use the same options attributes of function humanNumber(number, options = { decimals: 0, minDigits: 1 }) in formatters.js.

Reply:
Yes. that was Just a remark that `<`field name="my_integer_field" options="{'decimals': 3}" />``  will be weird.

But I have no alternative !

Reply:
I agree and I don't have a good idea. Maybe ask to Sad